internal/host/hostcommands/tang_connectivity_check_cmd.go

```diff
@@ -38,6 +38,11 @@ func (c *tangConnectivityCheckCmd) getTangServersFromHostIgnition(host *models.H
 		if err != nil {
 			return nil, err
 		}
+		if luks == nil || luks.Clevis == nil {
+			// TODO: not sure how this could happen or whether its possibly valid, for now pretend encryption is disabled just so we avoid a nil pointer dereference
+			c.log.Warn("luks configuration is missing or incomplete in the host ignition, disk encryption will be assumed to be disabled.")
```

Comment:
luks being nil is pretty common, it doesn't warrant a warning

Reply:
OK, removed the log entry.

Reply:
My comment was about only `luks` being `nil`, which doesn't warrant a warning.

But `luks` being non-nil and `Clevis` nil does warrant a warning, that's the case I've been discussing in all my comments so far, the one I don't understand

Reply:
OK, updated to deal with that scenario.